src/main/java/org/jenkinsci/plugins/GithubRequireOrganizationMembershipACL.java

```diff
@@ -102,6 +103,11 @@ public boolean hasPermission(@NonNull Authentication a, @NonNull Permission perm
                 return true;
             }
 
+            // Grant agent permissions to agent user
+            if (candidateName.equalsIgnoreCase(agentUserName) && checkAgentUserPermission(permission)) {
+                return true;
```

Comment:
Suggested for consistency with the existing approach:

```suggestion
                log.finest("Granting Agent Connect rights to user " + candidateName);
                return true;
```